src/code/providers/readonly-provider.coffee

```diff
@@ -51,7 +57,17 @@ class ReadOnlyProvider extends ProviderInterface
       # clone the metadata items so that any changes made to the filename or content in the edit is not cached
       callback null, _.map items, (metadataItem) -> new CloudMetadata metadataItem
 
-  canOpenSaved: -> false
```

Comment:
I think this needs to stay and be changed to ```-> true```.  The getHashParams method in the client uses is like ```if metadata?.provider?.canOpenSaved()```.  Maybe that should change to ```if metadata?.provider?.canOpenSaved?()```?